src/Knp/Menu/Twig/Helper.php

```diff
@@ -81,6 +86,99 @@ public function get($menu, array $path = array(), array $options = array())
      */
     public function render($menu, array $options = array(), $renderer =  null)
     {
+        $menu = $this->castMenu($menu);
+
+        return $this->rendererProvider->get($renderer)->render($menu, $options);
+    }
+
+    /**
+     * A string representation of this menu item
+     *
+     * e.g. Top Level > Second Level > This menu
+     *
+     * @param ItemInterface $item
+     * @param string        $separator
+     *
+     * @return string
+     */
+    public function pathAsString(ItemInterface $menu, $separator = ' > ')
+    {
+        if (null === $this->menuManipulator) {
+            throw new \BadMethodCallException('The menu manipulator must be set to transform a path to a string');
+        }
+
+        return $this->menuManipulator->getPathAsString($menu, $separator);
```

Comment:
this method could also be removed in favor of using the MenuManipulator directly in the extension as it just proxies the call.